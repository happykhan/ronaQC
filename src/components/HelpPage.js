import React from "react";
import { Typography, Container, Card, CardContent } from "@mui/material/";

const HelpPage = () => (
  <Container maxWidth="md">
    <Card>
      <CardContent>
        <Typography variant="h3" my={3} gutterBottom>
          Help
        </Typography>
        <Typography variant="body1">
          RonaQC accepts mapped SARS-CoV-2 reads (BAM format), generated from
          the SARS-CoV-2 bioinformatic pipelines like ARTIC, and any control
          samples from the respective sequencing run (negative/positive) as
          input. It will then assess the levels of cross contamination and
          primer contamination in the samples, and determine if the samples are
          reliable for detecting SARS-CoV-2, phylogenetic analysis, and/or
          submission to public databases.
        </Typography>
      </CardContent>
    </Card>
  </Container>
);
export default HelpPage;
