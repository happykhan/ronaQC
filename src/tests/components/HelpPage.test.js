/**
 * @jest-environment jsdom
 */
import React from "react";
import HelpPage from "../../components/HelpPage";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import { MemoryRouter } from "react-router-dom";

test("should render help page correctly", () => {
  render(
    <MemoryRouter>
      <HelpPage />
    </MemoryRouter>
  );
  expect(screen.getByTestId(`Help-page`)).toMatchSnapshot();
});
