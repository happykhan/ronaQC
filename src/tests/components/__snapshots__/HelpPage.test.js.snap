// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render help page correctly 1`] = `
<div
  class="MuiContainer-root MuiContainer-maxWidthMd css-xdmu94-MuiContainer-root"
  data-testid="Help-page"
>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <h3
        class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-pxxorf-MuiTypography-root"
      >
        Help
      </h3>
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        RonaQC accepts mapped SARS-CoV-2 reads (BAM format), generated from the SARS-CoV-2 bioinformatic pipelines like ARTIC, and any control samples from the respective sequencing run (negative/positive) as input. It will then assess the levels of cross contamination and primer contamination in the samples, and determine if the samples are reliable for detecting SARS-CoV-2, phylogenetic analysis, and/or submission to public databases.
      </p>
    </div>
  </div>
</div>
`;
